pyvista/plotting/camera.py

```diff
@@ -825,6 +830,17 @@ def tight(
         negative : bool, default: False
             Whether to view in opposite direction.
 
+        bounds : (x_min, x_max, y_min, y_max, z_min, z_max), optional
+            Manually specify bounds instead of calculating them from existing actors.
```

Comment:
The docs build failed, looks like we can't use the terms `x_min`, etc. here.

```suggestion
        bounds : sequence[float], optional
            Manually specify bounds instead of calculating them from existing actors.
            Bounds must have the form ``(x_min, x_max, y_min, y_max, z_min, z_max)``.
```